chatcommands.py

```diff
@@ -556,6 +556,43 @@ def approve(msg, pr_id):
     except Exception as e:
         raise CmdException(str(e))
 
+@command(str, privileged=True, whole_msg=True, give_name=True, aliases=["close", "reject-force", "close-force"])
+def reject(msg, args, alias_used="reject"):
+    argsraw = args.split(' "', 1)
+    try:
+        pr_id=int(argsraw[0].split(' ')[0])
+    except ValueError:
+        reason=''
+        pr_id=int(args.split(' ')[2])
+    try:
+        # Custom handle trailing quotation marks at the end of the custom reason, which could happen.
+        if argsraw[1][-1] == '"':
+            reason = argsraw[1][:-1]
+        else:
+            reason = argsraw[1]
+    except IndexError:
+        reason = ''
+    message_url = "https://chat.{0}/transcript/{1}?m={2}".format(msg._client.host, msg.room.id, msg.id)    
```

Comment:
Leave a blank line after the end of the `except` block:
```suggestion
    except IndexError:
        reason = ''

    message_url = "https://chat.{0}/transcript/{1}?m={2}".format(msg._client.host, msg.room.id, msg.id)    
```